src/coreclr/vm/methodtable.h

```diff
@@ -332,7 +332,9 @@ struct MethodTableAuxiliaryData
         enum_flag_CanCompareBitsOrUseFastGetHashCode       = 0x0004,     // Is any field type or sub field type overridden Equals or GetHashCode
 
         enum_flag_HasApproxParent           = 0x0010,
-        // enum_unused                      = 0x0020,
+#ifdef _DEBUG
+        enum_flag_IsPublished               = 0x0020,
+#endif
```

Comment:
What is the point of this publish logic?

Reply:
It exists so that we can write asserts that validate that any allocation of temporary entrypoints is properly tied to the right AllocMemTracker, so that cleanup can happen correctly. Adding these asserts has shaken out a distressing number of places I missed in my manual analysis.